cardano-c/Doxyfile

```diff
@@ -0,0 +1,2427 @@
+# Doxyfile 1.8.11
```

Comment:
can you regenerate the Doxyfile without the doc ?

Reply:
do you mean without the auto generated comments for every variable?